server/lib/handlebars.js

```diff
@@ -152,6 +152,30 @@ handlebars.registerHelper('encodeURIComponent', str => {
 
 handlebars.registerHelper('formatCurrencyObject', (obj, props) => formatCurrencyObject(obj, props.hash));
 
+handlebars.registerHelper('formatOrderSummary', order => {
```

Comment:
Something a bit more explicit there would be welcome. I can propose:

```suggestion
handlebars.registerHelper('formatOrderAmountWithInterval', order => {
```